### ISSUE TITLE
Exceptions when the chunk seems to be unloaded

### ISSUE THREAD

**Issue by guilhermepyva** [RS485/LogisticsPipes#1385]
When an extractor module is placed in an AE interface and the chunk is unloaded, this exception floods the console

```
> //
/* Exception occurred during a PhaseState */
//
/* Sponge's tracking system makes a best effort to not throw exceptions randomly */
/* but sometimes it is inevitable. In most cases, something else triggered this */
/* exception and Sponge prevented a crash by catching it. The following stacktrace */
/* can be used to help pinpoint the cause. */
//
/* The PhaseState having an exception: Tick{TileEntity} */
/* The PhaseContext: */
/* org.spongepowered.asm.util.PrettyPrinter@4bf0bc0a */
/* */
/* StackTrace : */
/* java.lang.IndexOutOfBoundsException: Index: 0, Size: 0 */
/* java.util.LinkedList.checkElementIndex(LinkedList.java:555) */
/* java.util.LinkedList.get(LinkedList.java:476) */
/* logisticspipes.proxy.specialinventoryhandler.AEInterfaceInventoryHandler.getStackInSlot(AEInterfaceInventoryHandler.java:233) */
/* logisticspipes.modules.ModuleExtractor.tick(ModuleExtractor.java:164) */
/* logisticspipes.modules.ChassiModule.tick(ChassiModule.java:137) */
/* logisticspipes.pipes.basic.CoreRoutedPipe.updateEntity(CoreRoutedPipe.java:444) */
/* logisticspipes.pipes.basic.LogisticsTileGenericPipe.func_73660_a(LogisticsTileGenericPipe.java:213) */
/* org.spongepowered.common.event.tracking.TrackingUtil.tickTileEntity(TrackingUtil.java:237) */
/* net.minecraft.world.WorldServer.updateTileEntity(WorldServer.java:4728) */
/* net.minecraft.world.WorldServer.redirect$onUpdateTileEntities$zlg000(WorldServer.java:4712) */
/* net.minecraft.world.World.func_72939_s(World.java:1835) */
/* net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:3930) */
/* net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:767) */
/* net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:397) */
/* net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668) */
/* net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526) */
/* java.lang.Thread.run(Thread.java:748) */
/* */
/* Minecraft : 1.12.2 */
/* SpongeAPI : 7.1.0-b38390e17 */
/* Sponge : 1.12.2-7.1.7-SNAPSHOT */
/* SpongeForge : 1.12.2-2838-7.1.7-RC3909 */
/* Minecraft Forge : 14.23.5.2838 */
```

**Comment by guilhermepyva** [comment]
Look like it happens when the chunk is loaded too

**Comment by bziemons** [comment]
Fixed in 86015c19e41781cd92c80b0a5b72946ca344a450
